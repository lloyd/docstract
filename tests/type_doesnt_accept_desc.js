/**
 * @type {string} no no, no desc on a @type
 */
exports.register = function() {
};
