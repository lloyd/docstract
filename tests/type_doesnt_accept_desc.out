{
  "args": [
    "Bogus arguments to @type: (2): {string} | no no, no ... at line 1"
  ], 
  "exception_type": "<type 'exceptions.RuntimeError'>", 
  "stack": [
    [
      "<module>", 
      "got = ds.extractFromFile(os.path.join(testDir, test + \".js\"))"
    ], 
    [
      "extractFromFile", 
      "data = self.extract(contents)"
    ], 
    [
      "extract", 
      "self._analyzeBlock(block, context, firstBlock, stack, lineStart, line)"
    ], 
    [
      "_analyzeBlock", 
      "self._consumeToks(tokens, parseData)"
    ], 
    [
      "_consumeToks", 
      "ctx = handler.parse(args)"
    ], 
    [
      "parse", 
      "(self.tagName, self._argPrint(args)))"
    ]
  ]
}
